PokeAlarm/WebhookStructs.py

```diff
@@ -114,6 +116,60 @@ def pokestop(data):
         return stop
 
     @staticmethod
+    def raid(data):
+        log.debug("Converting to raid: \n {}".format(data))
+
+        quick_id = check_for_none(int, data.get('move_1'), '?')
+        charge_id = check_for_none(int, data.get('move_2'), '?')
+
+        raid_end = None
+        raid_begin = None
+
+        if 'raid_begin' in data:
```

Comment:
Why are we treating raid and eggs as the same? While they have some similarities, Raids are essentially Pokemon with extra steps while eggs have far less variables. 

I think a much cleaner way to handle this would be to treat eggs and raids as different types of alerts. We need to detect which it is and construct a different struct for each. Each should be handled differently throughout the rest of the process. You did it this way for the alarms (raid vs raid_alert), but I think it needs to go a step farther and start here.